minari/dataset/minari_dataset.py

```diff
@@ -42,30 +43,55 @@ def parse_dataset_id(dataset_id: str) -> tuple[str | None, str, int | None]:
     return env_name, dataset_name, version
 
 
-def clear_episode_buffer(episode_buffer: Dict, eps_group: h5py.Group) -> h5py.Group:
+def clear_episode_buffer(episode_buffer: Dict, episode_group: h5py.Group) -> h5py.Group:
```

Comment:
Again, refactor needed to address in another PR. 
This code is extremely similar to the clear_buffer in data_collector

Reply:
sounds good, I'll make a task to refactor at the lines you mentioned.